### ISSUE TITLE
Menu cutting off on an iOS 6 device

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-iOS#238]
Testing again to reproduce..

![image uploaded from ios 1](https://cloud.githubusercontent.com/assets/1434597/22987067/3a271fe4-f373-11e6-9e0a-8a7c2437b3b7.jpg)

**Comment by chrisballinger** [comment]
I hope you mean iOS 8 :)

**Comment by cstiens** [comment]
He has an iPhone 6 running os version 10.1.1

**Comment by N-Pex** [comment]
I can't repro this in the simulator. Maybe only happens on a real device.

**Comment by cstiens** [comment]
Possibly. I'm having him test the latest version to see if it happens again.

**Comment by chrisballinger** [comment]
This could happen when the active call / location / audio / tethering / etc
"top bar" is on the screen, and then disappears.

**Comment by cstiens** [comment]
@chrisballinger You're right. This is what was reported, "I noticed that the menu doesn't get cut off as long as I'm only using Zom. But if I am using Zom while talking on the phone via Whatsapp or normal phone call, then the menu gets cut off at the bottom."

Is there anything we can do about this?
